tachyon/__main__.py

```diff
@@ -145,31 +141,33 @@ def finish_output(print_worker):
         pass
 
 
-def main():
-    # Parse command line
-    from tachyon.core.arguments import generate_options, parse_args
-
-    parser = generate_options()
-    options, args = parse_args(parser, sys.argv)
-
-    if not conf.eval_output and not conf.json_output:
-        print_program_header()
-
-    if len(sys.argv) <= 1:
-        parser.print_help()
-        print('')
-        return
+@click.command()
+@click.option("--cookie-file", default="")
+@click.option("--depth-limit", default=2)
+@click.option("--directories-only", is_flag=True)
+@click.option("--files-only", is_flag=True)
+@click.option("--json-output", is_flag=True)
```

Comment:
Missing short versions?

Reply:
I don't like non-descriptive options such as -f, -d, or -l, but I can add them if it is required.